### ISSUE TITLE
Campaign Financial Reports

### ISSUE THREAD

**Issue by yoanamanolova** [podkrepi-bg/frontend#1348]
### Which area(s) of Podkrepi.bg are affected? (leave empty if unsure)

Campaigns

### Describe the feature you'd like to request

Ability for organizers to attach financial reports to campaigns.

### Describe the solution you'd like

Release 1: Ability for organizers to attach financial reports and group them by spending/expense types
Release 2: Include visual graphic of spending/expenses by type
Release 3: Ability for organizers to attach and see campaign budget, including budgeted cost by spending/expense type
Release 4: Ability to see spending against budgeted costs

**Comment by swolf86** [comment]
![image](https://user-images.githubusercontent.com/75036745/236704296-00652d76-ed94-4841-93e9-fe8c6513aed4.png)

As a donor I want to see how the reports are formed - types of expenses how much, document as a link and preview as well as pie chart how it is all coming together so that I am aware of what is going on with my donation

**Comment by slavcho** [comment]
The things that are still pending on this ticket (see the last design above):
1. Pie chart
2. Icons assigned to the expenses types
3. Attached documents preview.

**Comment by yoanamanolova** [comment]
New ticket opened for outstanding items. Refer to ticket #1556
